dev/scripts/update-versions.sh

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bash
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+# This script is designed so that someone can update all versions for releases
+# using a single command.
+
+# Debugging
+# set -x
+set -e # exit if a command fails
+set -u # unbound variable is an error
+
+HELP=$(cat <<EOF
+Usage: update-version.sh [-s] <old version> <new version>
+
+    <old version>    The version to replace throughout the codebase
```

Comment:
can you provide an example version string?

Reply:
updated